src/Components/CreateImageWizard/formComponents/FileSystemConfiguration.js

```diff
@@ -120,6 +120,7 @@ const FileSystemConfiguration = ({ ...props }) => {
         unit: r.unit,
       });
       newOrder.push(id);
+      return null;
```

Comment:
I think this fix should be addressed a bit differently.

See, map is supposed to be used to transform a list into another one (check out the documentation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).

However here the return value of the result of the map function at line 113 is ignored. Meaning that what we want to have instead is a loop over all the entries of `fsc`.

So a better fix is to change `map` to `foreach` (check out the documentation there https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)

Does my explication makes sense?